plugins/modules/ec2_spot_instance.py

```diff
@@ -278,6 +278,13 @@
       - List of strings with IDs of spot requests to be cancelled
     type: list
     elements: str
+  terminate_instances:
+    description:
+      - Boolean value to set whether or not to terminate instances associated to spot request.
+      - Can be used only when I(state=absent).
+    default: False
+    type: bool
+    version_added: 5.3.0
```

Comment:
amazon.aws 5.3.0 has been release yesterday. I guess the next release is amazon.aws 6.0.0. Can you please update version_added?

@tremble Please, correct me if I am wrong.

Reply:
Since we've not released 6.0.0 I don't see a problem backporting this to stable-5 (5.4.0 at this point).  We can always release a final "feature" release of stable 5 at the same time we're preparing 6.0.0.

However, while I'm opinionated, this collection is technically maintained by your team, if you'd rather jump straight to 6.0.0 I don't feel strongly enough about it to push back.

Reply:
@tremble I recalled planning to release 6.0.0 somewhere in mid-April, but it was actually going to be somewhere by the end of April or beginning of May. By then we should have enough time and content for a potential 5.4.0 release, I guess.

So @mandar242, I guess you can go with `version_added: 5.4.0`

Reply:
Added.